src/service/service_core.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package helium;
+
+enum gateway_staking_mode {
+  dataonly = 0;
+  full = 1;
+  light = 2;
+}
+
+enum data_rate {
+    SF12BW125 = 0;
+    SF11BW125 = 1;
+    SF10BW125 = 2;
+    SF9BW125 = 3;
+    SF8BW125 = 4;
+    SF7BW125 = 5;
+    SF12BW250 = 6;
+    SF11BW250 = 7;
+    SF10BW250 = 8;
+    SF9BW250 = 9;
+    SF8BW250 = 10;
+    SF7BW250 = 11;
+    SF12BW500 = 12;
+    SF11BW500 = 13;
+    SF10BW500 = 14;
+    SF9BW500 = 15;
+    SF8BW500 = 16;
+    SF7BW500 = 17;
+    LRFHSS1BW137 = 18;
+    LRFHSS2BW137 = 19;
+    LRFHSS1BW336 = 20;
+    LRFHSS2BW336 = 21;
+    LRFHSS1BW1523 = 22;
+    LRFHSS2BW1523 = 23;
+    FSK50 = 24;
+    RFU = 25;
```

Comment:
Remove RFU.. Reserver Future Use is not something we need to do in an enum

Reply:
it was requested by mike above